pkg/controller/che/che_controller.go

```diff
@@ -608,6 +609,17 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 			}
 		}
 	}
+
+	// create ConsoleLink
+	spec := v1.ConsoleLinkSpec{Link: v1.Link{Href: "https://" + instance.Spec.Server.CheHost, Text: "Eclipse Che"},
+		Location: v1.ApplicationMenu,
+		ApplicationMenu: &v1.ApplicationMenuSpec{Section: "Red Hat Applications",
+			ImageURL: "https://raw.githubusercontent.com/eclipse/che-website/master/che/images/ico/96x96.png"}}
```

Comment:
what about
- https://{che_host}/dashboard/assets/branding/loader.svg
- https://{che_host}/dashboard/assets/branding/che-logo.svg
?

Reply:
nice, I'll use that